skbio/metadata/_intersection.pyx

```diff
@@ -0,0 +1,513 @@
+"""
```

Comment:
Thanks @mortonjt! @RNAer, can you pull down the original bx-python code (and unit test code) and try those files? That way we can verify whether the code needs to change (preferably not).